src/EmbeddedVsix/EmbeddedVsix.csproj

```diff
@@ -139,7 +139,7 @@
     -->
     <!-- NB we are specifying an output path for the task assembly to avoid potentially file locking issues
 	     with the version of the same project used by Integration.Vsix.csproj when fetching the CFamily files -->
-    <Exec Command="&quot;$(MSBuildBinPath)\msbuild.exe&quot; ..\..\build\DownloadSonarPlugin\DownloadSonarPlugin.csproj /p:PluginUrl=$(PluginUrl) /p:TargetDirectory=&quot;$(TypeScriptTargetDirectory)&quot; /p:UseSharedCompilation=false /t:Restore;Build;EmbedTypeScriptFiles /p:OutputPath=bin\GetTypeScriptFiles\ /p:VsTargetVersion=$(VsTargetVersion)" LogStandardErrorAsError="true" IgnoreExitCode="false" />
+    <Exec Command="&quot;$(MSBuildBinPath)\msbuild.exe&quot; ..\..\build\DownloadSonarPlugin\DownloadSonarPlugin.csproj /p:PluginUrl=$(PluginUrl) /p:TargetDirectory=&quot;$(TypeScriptTargetDirectory)&quot; /p:UseSharedCompilation=false /t:Restore;Build;EmbedTypeScriptFiles /p:OutputPath=bin\GetTypeScriptFiles\ /p:VsTargetVersion=$(VsTargetVersion) /nr:false /p:BuildInParallel=false" LogStandardErrorAsError="true" IgnoreExitCode="false" />
```

Comment:
I think the `/nr:false` is the important part of change - it stops the MSBuild server from holding a file lock on our assembly that contains the custom task.

Setting `BuildInParallel=false` here only affects the build and execution of the `DownloadSonarPlugin` project - it won't stop targets being executed in parallel for the rest of the SonarLint solution.